Libraries/ReactIOS/YellowBox.js

```diff
@@ -20,6 +20,7 @@ const StyleSheet = require('StyleSheet');
 
 const _warningEmitter = new EventEmitter();
 const _warningMap = new Map();
+const parseErrorStack = require('parseErrorStack');
```

Comment:
nit: This should be before the constant values (`_warningMap`, `_warningEmitter`), but with an extra line between it and the other modules because it is a pure function rather than a class.